packages/regl-worldview/src/commands/Command.js

```diff
@@ -81,6 +82,37 @@ export default class Command<T> extends React.Component<Props<T>> {
     }
   }
 
+  getDrawPropFromHitmapId(id: number) {
+    const { hitmapProps } = this.props;
+    for (const drawProp of hitmapProps) {
+      if (drawProp.color) {
+        const drawPropId = getIdFromColor(drawProp.color.map((color) => color * 255));
+        if (drawPropId === id) {
+          return drawProp;
+        }
+      }
+    }
+    return null;
+  }
+
+  fireRelevantMouseHandler(id: number, e: any, ray: any, mouseEventName: MouseEventEnum) {
```

Comment:
nit: would be nice to stay consistent with variable names between functions, e.g. `objectId` instead of just `id`

naming: maybe `handleMouseEvent` or something?